aspnetcore/tutorials/first-mvc-app/adding-model.md

```diff
@@ -57,15 +56,21 @@ The <xref:System.ComponentModel.DataAnnotations.DataType> attribute on `ReleaseD
 
 From the **Tools** menu, select **NuGet Package Manager** > **Package Manager Console** (PMC).
 
-![PMC menu](~/tutorials/first-mvc-app/adding-model/_static/pmc.png)
+![PMC menu](~/tutorials/first-mvc-app/adding-model/_static/pmc5.png)
+
+<!-- When https://dev.azure.com/devdiv/DevDiv/_workitems/edit/1320544 is fixed, we can remove the following install package instruction for Microsoft.EntityFrameworkCore.Design  -->
 
 In the PMC, run the following command:
 
 ```powershell
-Install-Package Microsoft.EntityFrameworkCore.SqlServer
+Install-Package Microsoft.EntityFrameworkCore.Design
```

Comment:
Why is this step missing from https://docs.microsoft.com/en-us/aspnet/core/tutorials/razor-pages/model ?

Reply:
Maybe related to the version available at the time?  I'm not sure.  I can't compile and run this one without it.  We will have to create an issue for the Razor-pages tutorial if it is a problem.